src/api/comments.ts

```diff
@@ -0,0 +1,16 @@
+import { client } from '../utils/fetchClient';
+import { Comment } from '../types/Comment';
+
+export const getPostComments = async (postId: number) => {
+  return Promise.resolve(await client.get<Comment[]>(`/comments?postId=${postId}`));
```

Comment:
fix, please, according to comment
![image](https://user-images.githubusercontent.com/59209337/205596448-c374d00c-5f1f-41d8-8c23-8c00869cf61d.png)